test/tests/disclosure-navigation.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+const { ariaTest } = require('..');
+const { By, Key } = require('selenium-webdriver');
+const assertAriaControls = require('../util/assertAriaControls');
+const assertAttributeValues = require('../util/assertAttributeValues');
+const assertTabOrder = require('../util/assertTabOrder');
+const assertHasFocus = require('../util/assertHasFocus');
+
+const exampleFile = 'disclosure/disclosure-navigation.html';
+
+const ex = {
+  buttonSelector: '#exTest button',
+  menuSelector: '#exTest > li > ul',
+  buttonSelectors: [
+    '#exTest > li:nth-child(1) button',
+    '#exTest > li:nth-child(2) button',
+    '#exTest > li:nth-child(3) button'
+  ],
+  menuSelectors: [
+    '#exTest > li:nth-child(1) ul',
+    '#exTest > li:nth-child(2) ul',
+    '#exTest > li:nth-child(3) ul'
+  ]
+};
+
+// Attributes
+
+ariaTest('"aria-controls" attribute on button', exampleFile, 'button-aria-controls', async (t) => {
+  t.plan(1);
+  await assertAriaControls(t, ex.buttonSelectors[0]);

```

Comment:
if you pass `ex.buttonSelector` instead of `ex.buttonSelectors[0]`, then `assertAriaControls` will test whether `aria-controls` is set on all menu button elements. Right now, it only tests whether `aria-controls` is on the first one.